src/Cleaners.py

```diff
@@ -0,0 +1,80 @@
+import re, pickle, os
```

Comment:
Your outline of the vignette looks good! Are you going to use a GUI or is it a program run from the command line? If it is a GUI, I would recommend putting screenshots of the interface in your vignette, if the latter, I would add code to let the user know how to run the functions. 

Regarding your code, I would add docstrings and comments to make it more understandable for others. I now had a hard time figuring out what the code does. But good idea to add errors when the input is not correct!